examples/run_loop_tool_sweep.py

```diff
@@ -0,0 +1,46 @@
+import sys
+
+import compiler_gym
+
+names = ["toggle_mode", "up", "down", "toggle_thread"]
+actions = [3, 0, 1, 3, 0]
+base = 1024 * 512
+K = int(sys.argv[1]) * base
+
+vectorize = int(sys.argv[2])
+run_log = int(sys.argv[3])
+env = compiler_gym.make("loop_tool-v0")
```

Comment:
Refactor this to `with compiler_gym.make(...) as env:` to ensure that `env.close()` is called